app/views/approval/ApprovalView.js

```diff
@@ -1,12 +1,20 @@
 import React, { Component } from 'react';
-import VolunteerApproval from '../../components/VolunteerApproval/VolunteerApproval'
+import './approview.css';
+import VolunteerApprovalContainer from '../../containers/VolunteerApprovalContainer/VolunteerApprovalContainer';
 import {Link} from 'react-router';
 
 class ApprovalView extends Component {
   render() {
       return (
           <div id="outer">
-            <VolunteerApproval />
+          	<div id= "volunteer">
+          	    <h1> Volunteer Approval </h1> 
+            	<VolunteerApprovalContainer />
+            </div> 
+            <div id ="hours">
+            <h1> Hour Approval </h1> 
+           		HOUR APPROVAL GOES HERE
```

Comment:
Anything that has to be completed at some later point, comment as `//TODO - comment about it`. This way we can easily search the entire project for missing/incomplete things